utils/apiFunctional.cjs

```diff
@@ -108,16 +109,19 @@ const trigger = function(gatewayUrl, username, password, team_name, project_name
 
 //method to check the execution status
 function checkExecStatus (host_name, port, testRunResponseBody, testSuiteName, emailId) {
+    //console.log(host_name, port, testRunResponseBody, testSuiteName, emailId);
     let apiCallConfig = {
         host: host_name,
         port: port,
         path: baseContext+'/checkExecutionStatus',
         method: 'POST',
         headers: {
             'Content-Type': 'application/json'
-        }
+        },
+        rejectUnauthorized: false
```

Comment:
why is this flag used?

Reply:
was getting self signed certificate in certificate chain error when ever we try to trigger Unum instance , so we used it